frame/assets/src/functions.rs

```diff
@@ -620,6 +620,55 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Ok((credit, source_died))
 	}
 
+	/// Issue a new class of fungible assets from a public origin.
+	///
+	/// This new asset class has no assets initially and its owner is the origin.
+	///
+	/// The origin must be Signed and the sender must have sufficient funds free.
+	///
+	/// Funds of sender are reserved by `AssetDeposit`.
+	/// Parameters:
+	/// - `id`: The identifier of the new asset. This must not be currently in use to identify
+	/// an existing asset.
+	/// - `admin`: The admin of this class of assets. The admin is the initial address of each
+	/// member of the asset class's admin team.
+	/// - `min_balance`: The minimum balance of this new asset that any single account must
+	/// have. If an account's balance is reduced below this, then it collapses to zero.
+	///
+	/// Emits `Created` event when successful.
+	pub(super) fn do_create(
```

Comment:
pub(super) does not make sense if you are trying to expose these functions to external crates. Should just be `pub`

Reply:
Yes; in fact, most of the functions in this file needs to be changed to `pub` when the intention here is to expose them as a public API.